tests/config_test.sh

```diff
@@ -129,6 +129,26 @@ function test_set_config_with_a_path_as_value()
   assert_equals_helper 'Change llvm' "($LINENO)" "$output" 'qemu_path_image=/DATA/QEMU_VMS/virty.qcow2'
 }
 
+function test_set_config_with_verbose_mode()
+{
+  local option
+  local new_value
+  local config_path
+  local output
+  local expected
+
+  option="default_to_recipients"
+  new_value="test@email.com"
+  config_path="${KW_CONFIG_BASE_PATH}/mail.config"
+  expected+="sed --in-place --regexp-extended --follow-symlinks \"s<(default_to_recipients=).*<\1test@email.com<\" \"${config_path}\""
+  expected+=$'\n'
+  expected+="sed --in-place --regexp-extended --follow-symlinks \"s<\#\s*default_to_recipients<default_to_recipients<\" \"${config_path}\""
+
+  output="$(set_config_value "$option" "$new_value" "$config_path" "VERBOSE")"
```

Comment:
Hey, two things:

1. I mislead you guys on my comment requesting this test case suggesting that you used `VERBOSE`. However, here you can use `TEST_MODE` instead, which makes `cmd_manager` output the command, but without actually running it. As we are testing the verbose aspect, the evaluation of the command is unnecessary and (not in this case) may add some significant time overhead to the test suite. Anyways, sorry for the slip 😅 
2. Use single-quotes around `VERBOSE` instead of double-quotes.

To not block you guys and as the mistake was mine, I fixed this myself.